recipes/mosquitto/epl-v20

```diff
@@ -0,0 +1,277 @@
+Eclipse Public License - v 2.0
```

Comment:
This is in the source tarball so you don't need it here (you still need the reference in the meta.yaml)